Jenkinsfile

```diff
@@ -14,8 +18,22 @@ pipeline {
             environment {
                 // This env variable avoids the "invalid container name" issue
                 COMPOSE_PROJECT_NAME = "${env.BUILD_TAG}"
+
+                // Since a successful run relies on environment varibles being set,
+                // we need to skip it for now.
+                SKIP_DRY_RUN="True"
+
+                CANVAS_API_URL = credentials('CANVAS_API_URL')
+                CANVAS_API_KEY = credentials('CANVAS_API_KEY')
+                KOPPS_BASE_URL = credentials('KOPPS_BASE_URL')
+                UG_URL = credentials('UG_URL')
+                UG_USERNAME = credentials('UG_USERNAME')
+                UG_PWD = credentials('UG_PWD')
+                UG_LDAP_BASE = credentials('UG_LDAP_BASE')
+                CSV_DIR = credentials('CSV_DIR')
```

Comment:
If the DRY_RUN is going to be skipped, should the env. variables be deleted?

Reply:
I guess it would mostly be to make sure integration tests could run properly, but there are no proper implementations of them in this project anyways. I'll remove these ones!